backbone.layoutmanager.js

```diff
@@ -262,11 +262,6 @@ var LayoutManager = Backbone.View.extend({
     return this.__manager__.renderDeferred.promise();
   },
 
-  // Proxy `then` for easier invocation.
-  then: function() {
-    return this.promise().then.apply(this, arguments);
```

Comment:
Ah. Setting `this` in the promise is quite handy though, more useful imo than avoiding a single method-call (`.promise()`).